packages/test/test-end-to-end-tests/src/test/gc/gcInactiveNodes.spec.ts

```diff
@@ -749,5 +758,89 @@ describeCompat("GC inactive nodes tests", "NoCompat", (getTestObjectProvider, ap
 				);
 			},
 		);
+
+		it("Reviving an InactiveObject clears Inactive state immediately in interactive client (but not for its subtree)", async () => {
+			const container1 = mainContainer;
+			const { summarizer: summarizer1 } = await createSummarizer(
+				provider,
+				container1,
+				testContainerConfig,
+			);
+			const defaultDataObject1 = (await container1.getEntryPoint()) as ITestDataObject;
+			await waitForContainerConnection(container1);
+
+			const dataObjectA_1 = await createNewDataObject();
+			const dataObjectB_1 = await createNewDataObject();
+			const idA = dataObjectA_1._context.id;
+			const idB = dataObjectB_1._context.id;
+
+			// Reference A from the container entrypoint, and B from A
+			defaultDataObject1._root.set("A", dataObjectA_1.handle);
```

Comment:
```suggestion
			defaultDataObject_1._root.set("A", dataObjectA_1.handle);
```
nit: To be consistent with other names.

Reply:
I only did `_1` when there was `A` or `B` in it. 🤷‍♂️

Reply:
(WF)